demo_drivers/bifurcation_tracking/periodic_orbit.cc

```diff
@@ -378,7 +378,8 @@ void ABCProblem<ELEMENT,TIMESTEPPER>::solve()
 {
  //Assign memory for the eigenvalues and eigenvectors
  Vector<std::complex<double> > eigenvalues;
- Vector< DoubleVector > eigenvectors;
+ Vector< DoubleVector > eigenvector_real;
```

Comment:
Slightly odd formatting. I wonder why `clang-format` hasn't removed the spaces inside the angle brackets...

Reply:
Good point. I've fixed it and we'll see what happens next time it goes in...